public/docs/_examples/structural-directives/dart/lib/unless_directive.dart

```diff
@@ -0,0 +1,31 @@
+// #docplaster
+// #docregion
+// #docregion unless-declaration
+import 'package:angular2/angular2.dart';
+// #enddocregion unless-declaration
+
+// #docregion unless-declaration
+@Directive(selector: '[myUnless]')
+class UnlessDirective {
+  // #enddocregion unless-declaration
+
+  // #docregion unless-constructor
+  TemplateRef _templateRef;
+  ViewContainerRef _viewContainer;
+
+  UnlessDirective(this._templateRef, this._viewContainer);
+  // #enddocregion unless-constructor
+
+  // #docregion unless-set
+  @Input() set myUnless(bool condition) {
```

Comment:
@kwalrath 
I did run dartfmt, but I put  `@Input` on purpose.
I'm also used to put it in its own line just like:
https://github.com/angular/angular.io/blob/master/public/docs/_examples/architecture/dart/lib/hero_detail_component.dart#L9
I've seen the annotation in the same line in other examples, like this one:
https://github.com/angular/angular.io/blob/master/public/docs/_examples/hierarchical-dependency-injection/dart/lib/hero_card_component.dart#L14

I also looked into the dart style guide but they dont say anything about it.
https://www.dartlang.org/effective-dart/style/